features/networking/multicast.feature

```diff
@@ -138,3 +138,135 @@ Feature: testing multicast scenarios
       | <%= cb.pod1ip %> : joined (S,G) = (*, 232.43.211.234), pinging |
       | <%= cb.pod1ip %> : multicast, xmt/rcv/%loss = 5/0/100%         |
 
+  # @author weliang@redhat.com
+  # @case_id OCP-12930
+  @admin
+  Scenario: Same multicast groups can be created in multiple tenant
+    Given I have a project
+    And evaluation of `project.name` is stored in the :proj1 clipboard
+    When I run the :create client command with:
+      | f | <%= BushSlicer::HOME %>/testdata/networking/multicast-rc.json |
+    Then the step should succeed
+    Given 3 pods become ready with labels:
+      | name=mcast-pods |
+    And evaluation of `pod(0).ip` is stored in the :proj1pod1ip clipboard
```

Comment:
@weliang1 how about just save the pod object itself instead of spawning a bunch of clipboards?  This will affect how you call the `cb` though.

Reply:
@pruan-rht Good suggestion, just wonder could you point me any existing scripts which use the way you mentioned above? So I can do the same way in this script. Thanks!

Reply:
@pruan-rht Could you point me any script example using the way you mentioned above? Thanks!

Reply:
@weliang1 `And evaluation of `pod.first` is stored in the :proj1_pod clipboard`

Reply:
@pruan-rht  The way of pod(0).ip, pod(0).name are used in existing multicast automation cases : https://github.com/openshift/verification-tests/blob/master/features/networking/multicast.feature#L17
To keep the multicast scripting consistent and easy to debug in the future, we can use same way as above scripts used, how do you think?

Reply:
I guess one other reason to store the whole pod is that we might want to start using `pod.ip_url` or other pod methods for automagic IPv6 support.

Reply:
@rbbratta We need write the new test cases for ipv6 multicast testing. I suggest to keep same scripting style as our existing multicast cases.